### PR TITLE
Untitled Pull Request

### PR DESCRIPTION
**Pull request by HappySwifter**
Trying to add calc functionality

### CODE REVIEW

File: Money/Views/CalculatorView.swift [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+//  Money
+//
+//  Created by Artem on 01.03.2024.
+//
+
+import SwiftUI
+
+enum CalculatorButtons: String, CaseIterable {
+    
+    case but7 = "7"
```
Comment (SibRinaT) [inline Money/Views/CalculatorView.swift:11]: its wrong!

**Review by SibRinaT — APPROVED** [review]
'APPROVE' performed by Xcode.

**Review by SibRinaT — APPROVED** [review]
'APPROVE' performed by Xcode.

File: Money/Views/CalculatorView.swift (hunk 1 above)
Comment (HappySwifter) [reply Money/Views/CalculatorView.swift:11]: it is OK!

**Pull request merged**